README.md

```diff
@@ -170,3 +170,110 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+#Deployment to Fly.io
+
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, 
+a command line interface that allows tou to do everything from creating an account to deploy to Fly.
+
+  - To install it on Linux
+
+    ```  
+      curl  -L https:fly.io/install.sh | sh`
+    ```
+
+- To install it on windows
+    ```
+      iwr https://fly.io/install.ps1 -useb
+    ```
+    
+    Visit `https://fly.io/` if you need installation guide
+  
+
+- Next authenticate with your fly.io account:
+    ```
+     fly auth login`
+    ```
+
+
+- To make sure everything is working well:
+    ```
+     fly apps list
+    ```
+  The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+  Environment Variables
+
+  We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+
+  Generate SECRET_KEY (if change needed)
+  - Run the following from the root of your project
+    ```
+    python3 generate_key.py
+    ```
+    Get your key and update in .env file SECRET_KEY='<your-key>'
+  
+
+  ### Deploy App
+
+  In this step the app is going to be launched to fly.io.
+  
+  - Create and configure new app
+    ```
+    fly launch
+    ```
+    
+  This command will create you an app on Fly.io , spin up a postgres instance, 
+  and create an app configuration named fly.toml in your project root. 
+  fly.toml file contains all app details.
+   
+  Copy the DATABASE_URL from the termial output of the above process(fly launch)
+    and Update DATABASE_URL in .env file.
+
+  
+- To make sure the app is created successfully:
+
+    ```
+     fly apps list
+    ```
+
+    This command prints 3 apps: 
+    1. your app
+    2. database instance and 
+    3. Fly builders: to build docker images
+
+#### Import Secrets
+
```

Comment:
Nope. Again this must be a script

Reply:
In fact the only manual intervention should be one script that asks your username. Yours would be 'robel'

Then it automatically generates the appname for you:

audio-discrimination-crowdsource-robel-prod

and


audio-discrimination-crowdsource-robel-staging